backend/test/test.js

```diff
@@ -1,54 +1,56 @@
 const chai = require("chai");
 const chaiHttp = require("chai-http");
-const io = require("socket.io-client");
+// const io = require("socket.io-client");
```

Comment:
Please cleanup all commented out code.

Reply:
I wish to fix the tests rather than removing them :D